piccolo/columns/column_types.py

```diff
@@ -2811,9 +2811,9 @@ def all(self, value: Any) -> Where:
         else:
             raise ValueError("Unrecognised engine type")
 
-    def cat(self, value: Union[Any, list[Any]]) -> QueryString:
+    def cat(self, value: list[Any]) -> QueryString:
```

Comment:
I changed this to `list[Any]` because `Union[Any, list[Any]]` is kind of weird because `Any` is so broad that it encapsulates `list[Any]`.

Also, we want people passing in lists. I've kept the logic for now which auto converts it back into a list, for backwards compatibility.